tests/cases/compiler/jsdocRestParameter.ts

```diff
@@ -0,0 +1,13 @@
+// @allowJs: true
+// @checkJs: true
+// @noEmit: true
+
+// @Filename: /a.js
+/** @param {...number} a */
+function f(a) {
+    // Ideally this would be a number. But currently checker.ts has only one `argumentsSymbol`.
+    arguments[0];
```

Comment:
Can you show off the type of a somehow, eg with assignment to another number variable?